lib/screens/topic_screen.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:flutter/material.dart';
+import 'package:maui/repos/article_repo.dart';
+import 'package:tuple/tuple.dart';
+
+class TopicScreen extends StatefulWidget {
+
+  TopicScreen({key}):super(key:key);
+  
+  @override
+  _TopicScreenState createState() => _TopicScreenState();
+}
+
+class _TopicScreenState extends State<TopicScreen> {
+
+  List<Tuple7<String, String, String, String, String, String, int >> _articles;
```

Comment:
Since there is already an Article class, no need to use Tuple. Tuples should be used only when you dont have a handy class to store the specific data.

Reply:
Okay sir I will look after it !